internal/handlers/users.go

```diff
@@ -23,7 +24,15 @@ type Users struct {
 }
 
 func NewUsers(us UsersService) Users {
-	return Users{}
+	return Users{user: us}
+}
+
+func (u Users) Routes() http.Handler {
```

Comment:
Is it consider to be a better practice to have routes configuration on handlers layers than directly in main.go ?

Reply:
Well. The idea is that we only decide with URI and which method should be used on that level. Remember when we were doing a unit we couldn't set properly the method and URI? Now we can improve the test and actually test POST, GET, etc requests. Meanwhile, on the main level, we decide on which subroute to attach this package, for example on `/users`.

Reply:
hm. I forget to add changes to the tests. I will do it in another PR